config/externalnamenottested.go

```diff
@@ -897,4 +897,107 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	"aws_lightsail_lb_https_redirection_policy": config.ParameterAsIdentifier("lb_name"),
 	// aws_lightsail_lb_stickiness_policy can be imported by using the lb_name attribute
 	"aws_lightsail_lb_stickiness_policy": config.ParameterAsIdentifier("lb_name"),
+
+	// macie2
+	//
+	// aws_macie2_classification_export_configuration can be imported using the account ID and region
+	"aws_macie2_classification_export_configuration": config.IdentifierFromProvider,
+
+	// medialive
+	//
+	// MediaLive Channel can be imported using the channel_id
+	"aws_medialive_channel": config.IdentifierFromProvider,
+	// MediaLive Input can be imported using the id
+	"aws_medialive_input": config.IdentifierFromProvider,
+	// MediaLive InputSecurityGroup can be imported using the id
+	"aws_medialive_input_security_group": config.IdentifierFromProvider,
+	// MediaLive Multiplex can be imported using the id
+	"aws_medialive_multiplex": config.IdentifierFromProvider,
+	// MediaLive MultiplexProgram can be imported using the id, or a combination of "program_name/multiplex_id"
+	"aws_medialive_multiplex_program": config.IdentifierFromProvider,
+
+	// msk
+	//
+	// MSK serverless clusters can be imported using the cluster arn
+	// Example: arn:aws:kafka:us-west-2:123456789012:cluster/example/279c0212-d057-4dba-9aa9-1c4e5a25bfc7-3
+	"aws_msk_serverless_cluster": config.TemplatedStringAsIdentifier("", "arn:aws:kafka:{{ .parameters.region }}:{{ .setup.configuration.account_id }}:cluster/{{ .parameters.cluster_name }}/{{ .external_name }}"),
+
+	// neptune
+	//
+	// aws_neptune_global_cluster can be imported by using the Global Cluster identifier
+	"aws_neptune_global_cluster": config.ParameterAsIdentifier("global_cluster_identifier"),
+
+	// networkfirewall
+	//
+	// Network Firewall Firewalls can be imported using their ARN
+	// Example: arn:aws:network-firewall:us-west-1:123456789012:firewall/example
+	"aws_networkfirewall_firewall": config.TemplatedStringAsIdentifier("name", "arn:aws:network-firewall:{{ .parameters.region }}:{{ .setup.configuration.account_id }}:firewall/{{ .external_name }}"),
```

Comment:
It seems that there is no region parameter in the resource schema. Could you please use the `.setup.configuration.region` instead of `.parameters.region`?

```suggestion
	"aws_networkfirewall_firewall": config.TemplatedStringAsIdentifier("name", "arn:aws:network-firewall:{{ .setup.configuration.region }}:{{ .setup.configuration.account_id }}:firewall/{{ .external_name }}"),
```